pages/common/make.md

```diff
@@ -0,0 +1,20 @@
+# make
+
+> Controls the compilation of an executable from source code.
```

Comment:
While this is true in 90% of cases, it is just a one case.
Generally speaking make is a task runner, similar to `ant`, `gulp`, `rake`.

I would probably say this as a first sentence, and then add that it is mostly used for compiling.

E.g. in `tldr` we also use make but for building.

So it can be something like this

```md
> Task runner for rules described in Makefile.
> Mostly used to controls the compilation of an executable from source code.
```